transaction/core/src/blockchain/block_version.rs

```diff
@@ -91,6 +91,11 @@ impl BlockVersion {
     pub fn validate_transaction_outputs_are_sorted(&self) -> bool {
```

Comment:
@jcape do you think there should be an MCIP mentioned here also? we could make a small MCIP mentioning that we want to enforce this

Reply:
Yeah, that's worth doing